### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by redbugz**
Bad dependencies and out of date files were causing builds depending on this to fail.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -16,14 +16,16 @@
...
     "angular-sanitize": "~1.2.32",
-    "hf": "git+https://github.com/fs-webdev/hf.git#2.4.5",
+    "hf": "fs-webdev/hf#~2.18.4",
     "inject": "^0.7.2",
     "jquery-ui": "jqueryui#1.10.4"
   },
-  "private": true
+  "private": true,
+  "resolutions": {
+    "fs-styles": "^1.x.x"
```
Comment (SirJackovich) [inline bower.json]: Should this be updated to 5.x.x?

**Comment by redbugz** [comment]
The .babelrc only matters if you have .es6 files which I don't think you have, and both changes are supposed to behave exactly like the old changes, they are just renaming from old deprecated names to the new names.

**Comment by redbugz** [comment]
The HF and fs-styles dependencies are both likely to be more impacting, but most likely they will be overridden by the app that is consuming this component, especially fs-styles v1 which is extremely old. I don't particularly care what the resolution is here, I would tend to use the latest, but I don't know what the downstream apps are using, and that's the version you should likely use.

**Comment by rosemariesadler** [comment]
@redbugz or @SirJackovich feel free to merge when you are ready.

**Pull request merged**